rules/signing.bzl

```diff
@@ -948,3 +948,46 @@ keyset = rule(
         ),
     },
 )
+
+def _signature_test_impl(ctx):
+    _script = ctx.actions.declare_file("{}.bash".format(ctx.label.name))
+
+    files = [f.short_path for f in ctx.files.srcs]
+    joined = " ".join(files)
+    ctx.actions.expand_template(
+      template = ctx.file._script,
+      output = _script,
+      substitutions = {
+        "@FILES@": joined,
```

Comment:
I'd just write `"@FILES@": " ".join(files),` here instead of creating a temporary.